### DISCUSSION TITLE
installation stops in themes patcher

### DISCUSSION THREAD

**Discussion by Madarastion** [prasanthrangan/hyprdots#1891] [Q&A]
![Screenshot 2024-09-28 045205](https://github.com/user-attachments/assets/77ad93c8-a39b-4a3a-b95e-b5d399d0d1e4)
it stops in themes patcher and nothing happens. at first i thought installation is finished so i rebooted the system but nothing happens.anyone know how to solve that issue ? this is on hyper-v by the way

**Comment by kRHYME7** [comment]
Try using `Hyde install`
